sirepo/auth/__init__.py

```diff
@@ -73,6 +74,11 @@
 #: visible_methods excluding guest
 non_guest_methods = None
 
+#: sirepo.uri_router module
```

Comment:
i think it is clear it's a module by it's name. rather, say "avoid circular import issues by importing in init_apis".

Reply:
c1482e38